charts/netbird/templates/dashboard-deployment.yaml

```diff
@@ -30,7 +30,7 @@ spec:
         {{- toYaml .Values.dashboard.podSecurityContext | nindent 8 }}
       containers:
         - name: {{ .Chart.Name }}
-          {{- if hasKey .Values.dashboard.podCommand "args" }}
+          {{- if and (hasKey .Values.dashboard.podCommand "args") .Values.dashboard.podCommand.args }}
```

Comment:
Can you add a bit of context about this change? Is some case where it was failing?

Reply:
I realised we don't need the `if and`. The desired result can be achieved using only an `if`.